### PR TITLE
Add information about the font for image call-outs and about creating xrefs in files

### PR DESCRIPTION
**Pull request by srpearce**
@mhoeber or @catong Please review these updates to the edX style guide.

### CODE REVIEW

**Comment by srpearce** [comment]
@lamagnifica @catong @pdesjardins Just found this old PR that makes some quick changes to the style guide. Would one of you please review it? Thanks!

File: en_us/edx_style_guide/source/ExampleRSTFile.rst [hunk 1]
```diff
@@ -353,10 +353,40 @@ is your preferred link text.
...
-::
+To reference an external web page, you add information to the file where you
+want the link, and then add an entry for the link to the links.rst file.
+
+#. In the paragraph where you want the link, create an anchor that represents
+   the text that you want to use for the link.
+
+   ::
+
+      The edX engineering wiki `Release Pages`_ provide access to detailed
```
Comment (pdesjardins) [inline en_us/edx_style_guide/source/ExampleRSTFile.rst:366]: Maybe explain that this is an example?

**Comment by pdesjardins** [comment]
Looks good. :+1:

File: en_us/edx_style_guide/source/ExampleRSTFile.rst [hunk 2]
```diff
@@ -353,10 +353,40 @@ is your preferred link text.
...
+      .. include:: ../../links/links.rst
+
+   .. note::
+
+      The path to the links.rst file depends on the location of the file where
+      you are creating the link. For example, the path might be
+      ``../../../links/links.rst`` or ``../links/links.rst``.
+
+#. In the ``edx-documentation/en_us/links/links.rst`` file, add an entry for
+   the anchor and link.
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/ExampleRSTFile.rst]: link --> URL

File: en_us/edx_style_guide/source/ExampleRSTFile.rst [hunk 3]
```diff
@@ -353,10 +353,40 @@ is your preferred link text.
 Cross references to external web pages
 ============================================
 
-This example also includes specific link text before the URL.
-::
+To reference an external web page, you add information to the file where you
+want the link, and then add an entry for the link to the links.rst file.
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/ExampleRSTFile.rst]: we're relying heavily on the word link. I'd suggest using link only for the links.rst file, and instead using cross reference, URL, and like that...

maybe,
To reference an external web page, you decide what text to use for the cross reference, such as the title of the web page, and add it to the file using this `markup`_. Then, you add the same text to the links.rst file along with the actual URL.

File: en_us/edx_style_guide/source/images.rst [hunk 4]
```diff
@@ -8,7 +8,7 @@ Because the user interface can change rapidly and frequently, and because it is
 very expensive for translation teams to localize screen shots, the
 documentation team uses screen shot images sparingly.
 
-.. contents:: Section Contents 
+.. contents:: Section Contents
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/images.rst]: we decided that we weren't going to include "Section Contents" after all

File: en_us/edx_style_guide/source/images.rst [hunk 5]
```diff
@@ -230,14 +243,14 @@ Image Sizes
...
-.. note that this only seems to control size in HTML output, not in PDFs. 
+.. note that this only seems to control size in HTML output, not in PDFs.
 .. - Alison 25 Sept 2015
 
 .. code-block:: xml
 
   .. image:: Images/descriptive_image_name.png
        :width: 600
-       :alt: 
+       :alt:
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/images.rst]: we should have an alt here, or some text saying that the alt is required, in complete, punctuated sentences.

File: en_us/edx_style_guide/source/ExampleRSTFile.rst [hunk 6]
```diff
@@ -353,10 +353,40 @@ is your preferred link text.
 Cross references to external web pages
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/ExampleRSTFile.rst]: title capitalization?

**Comment by lamagnifica** [comment]
:+1:

**Pull request merged**